src/status_im2/contexts/chat/messages/content/audio/component_spec.cljs

```diff
@@ -0,0 +1,52 @@
+(ns status-im2.contexts.chat.messages.content.audio.component-spec
+  (:require [status-im2.contexts.chat.messages.content.audio.view :as audio-message]
+            [test-helpers.component :as h]
+            [react-native.audio-toolkit :as audio]
+            [re-frame.core :as re-frame]))
+
+(defonce message
```

Comment:
@briansztamfater, `defonce` is unnecessary in pure maps, since they never change.